src/radical/pilot/compute_pilot.py

```diff
@@ -75,6 +73,14 @@ def __init__(self, pmgr, descr):
         # pilot failures can trigger app termination
         self._exit_on_error = self._descr.get('exit_on_error')
 
+        # ensure uid is unique
+        if self._uid:
+            if not self._umgr.check_uid(self._uid):
```

Comment:
Does the pilot have a `_umgr` object? Pylint objects and I think it is correct:
```
78   19   E1101   Instance of 'ComputePilot' has no '_umgr' member; maybe '_pmgr'? (no-member)
```

Reply:
Also, why do we care about the Unit manager to verify the pilot's ID and not the pilot manager?

Reply:
Yeah, that's a cut'n'paste error, should be the pmgr of course, thanks!